modules/swagger-codegen/src/main/resources/rust/api.mustache

```diff
@@ -0,0 +1,96 @@
+{{>partial_header}}
+use std::rc::Rc;
+use std::borrow::Borrow;
+
+use hyper;
+use serde_json;
+use futures;
+use futures::{Future, Stream};
+
+use super::{Error, configuration};
+
+pub struct {{{classname}}}Impl<C: hyper::client::Connect> {
+    configuration: Rc<configuration::Configuration<C>>,
+}
+
+impl<C: hyper::client::Connect> {{{classname}}}Impl<C> {
```

Comment:
@quodlibetor do you mind submitting a PR with the suggested change (together with the fix to replace "Go" with "Rust")?